models/Asset.php

```diff
@@ -1783,18 +1783,16 @@ public function setScheduledTasks($scheduledTasks)
 
     private function saveScheduledTasks()
     {
-        $this->getScheduledTasks();
-        $this->getDao()->deleteAllTasks();
-
-        if (is_array($this->getScheduledTasks()) && count($this->getScheduledTasks()) > 0) {
-            foreach ($this->getScheduledTasks() as $task) {
-                $task->setId(null);
-                $task->setDao(null);
-                $task->setCid($this->getId());
-                $task->setCtype('asset');
-                $task->save();
-            }
-        }
+        $scheduledTasks = $this->getScheduledTasks();
```

Comment:
I think we should move the existing `ScheduledTasksTrait` to `Pimcore\Model\Element\Traits` and use it for all elements.

Reply:
See https://github.com/pimcore/pimcore/pull/10320/commits/1f548009cd3c1b1357eaab98667609b3270b226e